test/test_integration_cli.py

```diff
@@ -380,7 +385,130 @@ def test_https_delete_policy_doesnt_break_if_no_created_roles(self):
     def test_https_can_get_whoami(self):
         self.setup_cli_params({})
 
-        output = invoke_cli(self, self.cli_auth_params, ['whoami'])
+        output = self.invoke_cli(self.cli_auth_params, ['whoami'])
         response = json.loads(output)
-        self.assertEquals(response.get('account'), 'dev')
+        self.assertEquals(response.get('account'), f'{self.client_params.account}')
         self.assertEquals(response.get('username'), 'admin')
+
+# class CliIntegrationTestWithConjurLocalEnv(IntegratioTestCaseBase):
```

Comment:
what is this giant block of commented out test?

Reply:
removed